src/bindings/c/tests/ov_compiled_model_test.cpp

```diff
@@ -135,9 +135,14 @@ TEST_P(ov_compiled_model_test, get_property) {
     OV_EXPECT_OK(ov_core_compile_model(core, model, device_name.c_str(), 0, &compiled_model));
     EXPECT_NE(nullptr, compiled_model);
 
+    const char* auto_batch_timeout_set_property = ov_property_key_auto_batch_timeout;
+    OV_EXPECT_OK(ov_compiled_model_set_property(compiled_model, auto_batch_timeout_set_property));
```

Comment:
You need to give a value to auto_batch_timeout. I suggest to get_property firstly, and set_property to another different value, then get_property to verify it is set correctly.